<launch>
  <node name="superpixel_smoother_node" pkg="superpixel_clutter_smoother"
	type="superpixel_smoother_node" respawn="false" output="screen">
    <param name="superpixel_sigma" type="double" value="0.3"/>
    <param name="superpixel_k" type="int" value="500"/>
    <param name="min_patch_size" type="int" value="10"/>
  </node>

  <node name="offline_clutter_test_node" pkg="superpixel_clutter_smoother"
	type="offline_clutter_test_node" respawn="false" output="screen">
    <param name="input_image_name" type="string" value="/home/thermans/data/clutter_hrl/0-19/4.png"/>
    <param name="input_mask_name" type="string" value="/home/thermans/data/clutter_hrl/0-19/4_mask.png"/>
    <param name="input_pcd_name" type="string" value="/home/thermans/data/clutter_hrl/0-19/4.pcd"/>
    <param name="smooth_output_name" type="string" value="/home/thermans/data/smooth-output.png"/>
    <param name="clutter_output_name" type="string" value="/home/thermans/data/clutter-output.png"/>
  </node>

</launch>
